src/components/AgreementLineSections/Info/Info.js

```diff
@@ -45,208 +48,196 @@ const propTypes = {
-          createdDate={line.dateCreated}
-          hideSource
-          id="agreementLineInfoRecordMeta"
-          lastUpdatedDate={line.lastUpdated}
-        />
-        <KeyValue
-          label={<FormattedMessage id="ui-agreements.line.parentAgreement" />}
-        >
-          <div data-test-agreement-line-agreement>
-            {line.owner?.id ? (
-              <Link to={urls.agreementView(line.owner.id)}>
-                {line.owner.name}
-              </Link>
-            ) : (
-              <NoValue />
-            )}
-          </div>
-        </KeyValue>
-        <Row>
-          <Col md={3} xs={6}>
-            <KeyValue
-              label={
-                <FormattedMessage id="ui-agreements.eresources.activeFrom" />
-              }
-            >
-              <div data-test-agreement-line-active-from>
-                {line?.activeFrom ? (
-                  <FormattedUTCDate value={line?.activeFrom} />
-                ) : (
-                  <NoValue />
-                )}
-              </div>
-            </KeyValue>
-          </Col>
-          <Col md={3} xs={6}>
-            <KeyValue
-              label={
-                <FormattedMessage id="ui-agreements.eresources.activeTo" />
-              }
-            >
-              <div data-test-agreement-line-active-to>
-                {line?.activeTo ? (
-                  <FormattedUTCDate value={line?.activeTo} />
-                ) : (
-                  <NoValue />
-                )}
-              </div>
-            </KeyValue>
-          </Col>
-          {isSuppressFromDiscoveryEnabled('agreementLine') ? (
-            <Col md={3} xs={12}>
-              <KeyValue
-                label={
-                  <FormattedMessage id="ui-agreements.agreementLines.suppressFromDiscovery" />
-                }
-              >
-                <div data-test-agreement-line-suppress-from-discovery>
-                  <FormattedMessage
-                    id={`ui-agreements.${line.suppressFromDiscovery ? 'yes' : 'no'}`}
-                  />
-                </div>
-              </KeyValue>
-            </Col>
-          ) : null}
-          <Col md={3} xs={12}>
-            <KeyValue label={<FormattedMessage id="ui-agreements.note" />}>
-              <div data-test-agreement-line-note>
-                {line.note ?? <NoValue />}
-              </div>
-            </KeyValue>
-          </Col>
-        </Row>
-        <Row>
+    <KeyValue label={<FormattedMessage id="ui-agreements.eresources.proxiedAndCustomizedUrls" />}>
+      <MultiColumnList
+        columnMapping={{
+          name: <FormattedMessage id="ui-agreements.eresources.proxiesCustomizers" />,
+          url: <FormattedMessage id="ui-agreements.eresources.url" />,
+        }}
+        contentData={sorted}
+        id="templated-urls"
+        interactive={false}
+        visibleColumns={['name', 'url']}
+      />
+    </KeyValue>
+  );
+};
+
+const renderErrorCard = (resource) => {
```

Comment:
This should probably be a "ResourceErrorCard" component rather than a floating render function if it's out here to be reused.


If we don't reuse it then you can define it within the component itself (Same goes for the renderTemplatedUrls and getTitlePlatformUrl)

Reply:
Moved functions inside the component